core/dropdowndiv.js

```diff
@@ -185,10 +201,16 @@ Blockly.DropDownDiv.clearContent = function() {
  * Set the colour for the drop-down.
  * @param {string} backgroundColour Any CSS colour for the background.
  * @param {string} borderColour Any CSS colour for the border.
+ * @param {boolean=} opt_invertedText Whether or not the text color should be
```

Comment:
Why `opt_invertedText` instead of `opt_className`?

Reply:
I was hoping to just provide the two options, and that was a way to indicate whether or not the color needs inverted text or not.
I will leave this out of this PR though, and think about it a little more.